server/eof/fraValueService.ts

```diff
@@ -1,6 +1,7 @@
 import { getDynamicCountryConfiguration } from '@server/repository/country/getDynamicCountryConfiguration'
+import { readEofOdps } from '@server/repository/odp/readEofOdps'
+import { readFocOdps } from '@server/repository/odp/readFocOdps'
```

Comment:
we need to think about FraValueService. Mayne we should use OdpService in this case?